test-cypress/integration/sparql/sparql.menu.spec.js

```diff
@@ -1,807 +1,1113 @@
-describe('SPARQL screen validation', function () {
-    let repoId = 'sparqlRepo' + Date.now();
+import ImportSteps from '../../steps/import-steps';
 
-    let modifiedDefaultQuery = 'prefix spif: <http://spinrdf.org/spif#>\n' +
+describe('SPARQL screen validation', () => {
+    let repositoryId;
+
+    const DEFAULT_QUERY = 'select * where { \n' +
+        '\t?s ?p ?o .\n' +
+        '} limit 100';
+
+    const DEFAULT_QUERY_MODIFIED = 'prefix spif: <http://spinrdf.org/spif#>\n' +
         'select * {\n' +
-        '   ?x spif:for (1 2000)\n' +
+        '    ?x spif:for (1 2000)\n' +
         '} limit 1001';
 
-    let defaultQuery = 'select * where { \n' +
-        '\t?s ?p ?o .\n' +
-        '} limit 100 ';
+    function createRepoAndVisit(repoOptions = {}) {
+        repositoryId = 'sparql-' + Date.now();
+        repoOptions.id = repositoryId;
+        cy.createRepository(repoOptions);
+        cy.warmRepositoryNamespaces(repositoryId);
```

Comment:
Should we always wait for the namespaces after creating the repo?

Reply:
Not sure so that's why I didn't add it by default after each repo creation.